src/applications/mhv-secure-messaging/components/ComposeForm/ReplyDraftItem.jsx

```diff
@@ -59,6 +60,7 @@ const ReplyDraftItem = props => {
   const composeFormActionButtonsRef = useRef(null);
 
   const folderId = useSelector(state => state.sm.folders.folder?.folderId);
+  const isPilot = useSelector(state => isPilotState(state));
```

Comment:
same here.  We don't need to pass state to isPilotState.  useSelector(isPIlotState) should work